src/nt_users_tool/net_commands.py

```diff
@@ -0,0 +1,72 @@
+from os import popen
+from concurrent.futures import Executor, ThreadPoolExecutor
+
+from typing import NamedTuple
+
+from constants import LAST_RELEVANT_ELEMENT_POSITION
+
+class NTUserInfo(NamedTuple):
+    """Dataclass for a user of the bosch network
+
+    :param NamedTuple: Full name, nt_user and expiration date (fetched from net user /domain)
+    """
+    full_name: str
+    nt_user: str
+    expiration_date: str
+
+def get_nt_user_string(nt_user: str) -> str:
+    """Executes a net command with given nt_user and returns the response as a string.
+
+    :param nt_user: The nt_user that will be used in the net command.
+    :return: The response as a string.
+    """
+    input = nt_user.upper()
+    command = (f"net user /domain {input}")
```

Comment:
there is no need for brackets
```suggestion
    command = f"net user /domain {input}"
```

Reply:
Brackets removed.